### ISSUE TITLE
Implement custom graphics

### ISSUE THREAD

**Issue by DUDEbehindDUDE** [DUDEbehindDUDE/GPA-Galaxy#4]
Instead of just sticking to the material design, it would be cool to have this "galaxy" aesthetic where the background is like stars and stuff (this would further enhance the theme and stuff).

This is something very low priority; this doesn't need to happen until after most of the design and content is implemented.

**Comment by DUDEbehindDUDE** [comment]
Completed via 131d19bfabc80cf77c38afb5e8287dbc8df1ecab
